src/View/Widget/DateTimeWidget.php

```diff
@@ -188,6 +188,7 @@ protected function formatDateTime(DateTimeInterface|string|int|null $value, arra
 
         try {
             if ($value instanceof DateTimeInterface) {
+                /** @var \DateTime $dateTime */
```

Comment:
```suggestion
                /** @var \DateTime|\DateTimeImmutable $dateTime */
```
Can we remove the @psalm-suppress below now?